lib/puppet/functions/sort.rb

```diff
@@ -0,0 +1,74 @@
+# Sorts an Array or String lexicographically
```

Comment:
Should it really say "lexicographically" here given that it is possible sort in anyway desired using a block?

Reply:
Changed the documentation to say `Sorts an Array numerically or lexicographically or the characters of a String lexicographically.`